### PR TITLE
feat: enable blst portable by default

### PR DESCRIPTION
**Pull request by Rjected**
This is so we can enable the underlying blst `portable` feature in reth for creating portable release binaries

### CODE REVIEW

**Comment by jtraglia** [comment]
Hey @Rjected thanks for the PR. This might have actually been an oversight on our part. The other bindings are using the portable version of blst by default, with the exception of Go because we can't control it from here. Semi-recently, the authors of blst added "cpu runtime detection" to the portable version of blst; so if the system is using a new cpu it will use optimized instructions and if it's an older cpu it will use the portable implementation. This is a long way of saying we should be using portable by default.

cc @asn-d6 @pawanjay176

**Comment by jtraglia** [comment]
> Does this mean we should set `default = ["portable"]` or `default = ["blst/portable"]`?

Yes, and I slightly favor the second one but it's not a strong opinion.

**Comment by Rjected** [comment]
> > Does this mean we should set `default = ["portable"]` or `default = ["blst/portable"]`?
> 
> Yes, and I slightly favor the second one but it's not a strong opinion.

makes sense, I just changed it to the second one

**Review by jtraglia — APPROVED** [review]
Looks great, thanks again! I will wait for another maintainer to approve before merging.

**Pull request merged**
